src/main/resources/assets/computercraft/lang/de_DE.lang

```diff
@@ -0,0 +1,64 @@
+tile.computercraft:computer.name=Computer
+tile.computercraft:advanced_computer.name=Erweitertetr Computer
+tile.computercraft:drive.name=Diskettenlaufwerk
+tile.computercraft:printer.name=Drucker
+tile.computercraft:monitor.name=Monitor
+tile.computercraft:advanced_monitor.name=Erweiterter Monitor
+tile.computercraft:wireless_modem.name=WLAN-Modem
+tile.computercraft:wired_modem.name=LAN-Modem
+tile.computercraft:cable.name=LAN-Kabel
+tile.computercraft:command_computer.name=Befehlscomputer
+tile.computercraft:advanced_modem.name=Ender-Modem
+tile.computercraft:speaker.name=Lautsprecher
+
+tile.computercraft:turtle.name=Roboter
+tile.computercraft:turtle.upgraded.name=%s-Bot
+tile.computercraft:turtle.upgraded_twice.name=%s & %s-Bot
```

Comment:
This will clash with the adjectives in line 32 to 40.

Reply:
Mhh, yes there a few problems. But i think that this seems to be "incorrectable" since German has another..... dont know how to call it.

I'll try to fix that as good as possible.

Reply:
Your best option would be either to turn all adjectives into actual adjectives, probably.